api/db/drop.js

```diff
@@ -107,8 +117,19 @@ const dropFriendships = async (confirmed = false) => {
   }
 };
 
+const dropComments = async (confirmed = false) => {
+  console.log('\n', '\x1b[0m', 'Dropping comments collection...');
+  if (confirmed || (await getCommentsCount()) !== 0) {
+    await Comment.collection.drop();
+    console.log('\x1b[31m', 'Dropped comments collection');
+  } else {
+    console.log('\x1b[33m', 'Comments collection is already empty');
+  }
+};
+
```

Comment:
And same here: I think this should go above the `dropFriendships` function definition